test/src/unit-capi-group.cc

```diff
@@ -538,16 +538,16 @@ TEST_CASE_METHOD(
   REQUIRE(rc == TILEDB_OK);
 
   rc =
-      tiledb_group_add_member(ctx_, group1, array1_uri.c_str(), false, nullptr);
+      tiledb_group_add_member(ctx_, group1, array1_uri.c_str(), false, "one");
```

Comment:
Please add a new test section instead of changing this one in line. By adding names here there is no longer any non-serialization tests that test group members without names.

Reply:
reverted modified cases to original
created new cases with names